dwave/system/testing.py

```diff
@@ -104,7 +132,8 @@ def __init__(self, broken_nodes=None, broken_edges=None,
         properties['num_qubits'] = len(solver_graph)
         properties['category'] = 'qpu'
         properties['quota_conversion_rate'] = 1
-        properties['topology'] = {'type' : topology_type, 'shape' : topology_shape}
+        properties['topology'] = {'type' : topology_type,
+                                  'shape' : topology_shape}
```

Comment:
[Spacing](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements) around `:`.